setup.py

```diff
@@ -7,4 +7,8 @@
     version='1.0',
     description='Service to interact with users.',
     author='Grupo 5',
+    packages=['kubernetes'],
+    package_dir={
+    	'kubernetes': 'users'
+    }
```

Comment:
This will be adding the Kubernetes directory to the python package.  
You need to actually need to tell setuptools to ignore the directory with:

```suggestion
    packages=[''],
    include_package_data=True,
    exclude_package_data={'': ['tests', 'kubernetes']},
```

Reply:
FWI: If your local tox run fails, you can run `tox -r` the `-r` flag tells tox remove all cached data (dependencies/compiled code)